README.md

```diff
@@ -51,26 +36,16 @@ The Postgres Operator made it to the [Google Summer of Code 2019](https://summer
 * [cluster manifest reference](docs/reference/cluster_manifest.md)
 * [command-line options and environment variables](docs/reference/command_line_and_environment.md)
 
-The rest of this document is a tutorial to get you up and running locally with the operator on Minikube.
-
-## Overview of involved entities
```

Comment:
Where are the entities now?

Reply:
Moved to index.md as well. I thought it's a better place than the README, which is quite bloated right now.